spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SelectTable.scala

```diff
@@ -80,12 +80,12 @@ class SelectTable(val index: Int)(implicit ev: TensorNumeric[Float]) extends Mkl
 
   override private[mkldnn] def initFwdPrimitives(inputs: Array[MemoryData], phase: Phase) = {
     _inputFormats = inputs
-    _outputFormats = Array(inputs(index))
+    _outputFormats = Array(inputs(index - 1))
```

Comment:
double check

Reply:
Change it back now.

Reply:
For SelectTable, it requires the start index is 1. So it shoud be like this here.